app/lib/clients/bambooHR_client.rb

```diff
@@ -0,0 +1,45 @@
+require 'bamboozled'
+require 'date'
+
+
+class Clients::BambooHRClient
+    client = Bamboozled.client(subdomain: AppEnv['BAMBOOHR_SUBDOMAIN'], api_key: AppEnv['BAMBOOHR_API_KEY'])
+    # birthday = client.employee.all("birthday")
+    # anniversary = client.employee.all("hireDate")
+    day_before_yesterday = (Date.today - 2).to_formatted_s(:iso8601)
```

Comment:
Nit: we should probably start calculating things like today/tomorrow/yesterday based on a Location's timezone. In that case, we should probably pass a timezone to a method that uses that to calculate these values. Rails has some builtin methods for that: https://api.rubyonrails.org/v5.1.7/classes/ActiveSupport/TimeZone.html#method-i-yesterday